### ISSUE TITLE
grammarkdown emitter does not include blank lines between productions wrapped in `<ins>`

### ISSUE THREAD

**Issue by bakkot** [rbuckton/grammarkdown#80]
A simple reproduction:

```js
'use strict';
let { CoreAsyncHost, Grammar, GrammarkdownEmitter } = require('grammarkdown');

let source = `
X ::
  Y

<ins>
X ::
  Y
</ins>

X ::
  Y
`;

(async()=>{
  const grammarHost = CoreAsyncHost.forFile(source);
  const grammar = new Grammar([grammarHost.file], {}, grammarHost);
  await grammar.bind();
  (new GrammarkdownEmitter({})).emit(grammar.rootFiles[0], grammar.resolver, grammar.diagnostics, (file, result) => {
    console.log(result);
  });
})().catch(e => {
  console.error(e);
  process.exit(1);
});
```

This prints
```
X ::
    Y

<ins>X ::
    Y
</ins>
X ::
    Y
```

which seems suboptimal.

**Comment by rbuckton** [comment]
> `#id`'s aren't printed

Can you clarify what you expect to be printed?

**Comment by rbuckton** [comment]
Published in `grammarkdown@3.2.0`
